scribble/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CommentsController < ApplicationController
+  def index
+  end
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @post.comments.create!(comment_params)
+    redirect_to post_path(@post)
+  end
+
+  def edit
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+  end
+
+  def update
+    @comment = Comment.find(params[:id])
+    @comment.update(comment_params)
+    @post = Post.find(params[:post_id])
+    redirect_to @post
```

Comment:
you could replace these last two lines with something simpler:
```
redirect_to @comment.post
```

No need to find the post and store it in a var, when you can use the comment / post association methods. :)